docs/5/api-reference/directives.md

```diff
@@ -2432,6 +2432,13 @@ directive @paginate(
   """
   builder: String
 
+  """
```

Comment:
Update the `master` docs instead, see https://github.com/nuwave/lighthouse/blob/master/docs/.github/README.md#documenting-new-features

Reply:
Working on this after changing the master, wanted to revert v5 and run `yarn release` which I ran after yarn and yarn start, but it gives:

```
error Command "release" not found.
```

I checked package.json and there was no scripts for release indeed:

```json
{
    "license": "MIT",
    "dependencies": {
        "@vuepress/plugin-back-to-top": "1.8.2",
        "@vuepress/plugin-medium-zoom": "1.8.2",
        "fs-extra": "9.1.0",
        "vuepress": "1.8.2",
        "vuepress-theme-default-prefers-color-scheme": "1.0.9"
    },
    "devDependencies": {
        "prettier": "2.2.1"
    },
    "scripts": {
        "start": "vuepress dev .",
        "build": "vuepress build .",
        "prettify": "prettier --write --tab-width=2 **/**/*.md **/*.js .vuepress/*.js"
    }
}
```

Reply:
> Update the `master` docs instead, see https://github.com/nuwave/lighthouse/blob/master/docs/.github/README.md#documenting-new-features

Working on this after changing the master, wanted to revert v5 and run `yarn release` which I ran after yarn and yarn start, but it gives:

```
error Command "release" not found.
```

I checked package.json and there was no scripts for release indeed:

```json
{
    "license": "MIT",
    "dependencies": {
        "@vuepress/plugin-back-to-top": "1.8.2",
        "@vuepress/plugin-medium-zoom": "1.8.2",
        "fs-extra": "9.1.0",
        "vuepress": "1.8.2",
        "vuepress-theme-default-prefers-color-scheme": "1.0.9"
    },
    "devDependencies": {
        "prettier": "2.2.1"
    },
    "scripts": {
        "start": "vuepress dev .",
        "build": "vuepress build .",
        "prettify": "prettier --write --tab-width=2 **/**/*.md **/*.js .vuepress/*.js"
    }
}
```

Reply:
Sorry, it is `make release` now. I just fixed the docs.

Reply:
> Sorry, it is `make release` now. I just fixed the docs.

Thank you but is there still any alternatives for not using docker and releasing it?
I'm on MacOS and don't have docker. (Docker is terrible on MacOS...)

Reply:
`make release` requires `make`, not Docker. If you do not have that, just look into the makefile and see what it does - you will find it to be quite trivial 😉

Reply:
> `make release` requires `make`, not Docker. If you do not have that, just look into the makefile and see what it does - you will find it to be quite trivial 😉

Sorry got confused by :

```
If you use Docker you can start up the environment (including docs) by running:

make setup
make node
```

Thought I need these first to run the release, ran it without these and it worked thanks!